protocol/testutil/app/app.go

```diff
@@ -83,6 +83,11 @@ type AdvanceToBlockOptions struct {
 	// The time associated with the block. If left at the default value then block time will be left unchanged.
 	BlockTime time.Time
 
+	// Whether to increment the block time gradually and evenly among the advanced blocks.
+	// TODO(DEC-2156): Instead of an option, pass in a `BlockTimeFunc` to map each block to a
+	// time. This gives user more flexibility.
```

Comment:
```suggestion
	// TODO(DEC-2156): Instead of an option, pass in a `BlockTimeFunc` to map each block to a
	// time giving user greater flexibility.
```